packages/backend/src/request/request-context-provider.spec.ts

```diff
@@ -31,15 +31,18 @@ describe('RequestContextProvider', () => {
 
     let app = new App();
     let requestContextProvider = new RequestContextProvider(app);
-    requestContextProvider
+    return requestContextProvider
       .withRequestContext(request, response, _sctx => {
-        expect(() => {
-          requestContextProvider.withRequestContext(request, response, () => Promise.resolve());
-        }).toThrow();
-        return Promise.resolve();
+        return Promise.resolve(
```

Comment:
... but this one is not, since expect can only catch sync returns.

Reply:
The return is because `withRequestContext` expects a promise. Should I unwrap and move `return Promise.resolve();` at the end?

Reply:
yes that should make it clearer they are not related